text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/Context.java

```diff
@@ -37,19 +39,48 @@
 @ApiStatus.NonExtendable
 public interface Context {
   /**
-   * Returns original message as provided to the parser.
+   * Parses a MiniMessage using all the settings of this context, including placeholders.
```

Comment:
Probably don't need to specify placeholders here anymore right?

```suggestion
   * Parses a MiniMessage string using all the settings of this context.
```